regression/scripts/multifidelity/TRM_hist.txt

```diff
@@ -1,4 +1,4 @@
-2020-01-15 10:10:29.127957
+2020-01-16 11:05:23.431717
```

Comment:
This might be my fault, but can you remove these files (this and add_hist.txt)? I don't think they're needed in the code base.

Reply:
Files removed

Reply:
I may be misreading Github here, but they're still appearing as part of the PR for me. Is there a particular commit that was pushed to remove them?

Reply:
check the last commit I made